es-app/src/SystemData.cpp

```diff
@@ -148,6 +148,9 @@ void SystemData::indexAllGameFilters(const FileData* folder)
 		{
 			case GAME:   { mFilterIndex->addToIndex(*it); } break;
 			case FOLDER: { indexAllGameFilters(*it);      } break;
+			default:
```

Comment:
Added default case to resolve compiler warning:

/home/devel/temp/EmulationStation_Stable/es-app/src/SystemData.cpp:147:10: warning: enumeration value
 'PLACEHOLDER' not handled in switch [-Wswitch]
                switch((*it)->getType())
                       ^~~~~~~~~~~~~~~~
1 warning generated.